test/integration/contact_system.cc

```diff
@@ -100,6 +100,12 @@ TEST_F(ContactSystemTest,
       EXPECT_NEAR(0.25, std::abs(contact.position(0).x()), 5e-2);
       EXPECT_NEAR(1, std::abs(contact.position(0).y()), 5e-2);
       EXPECT_NEAR(1, contact.position(0).z(), 5e-2);
+      bool EntityNameFound =
```

Comment:
nit, capitalization: 
```suggestion
      bool entityNameFound =
```

Reply:
Done